src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  const values = [];
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (values.includes(value)) {
+      return null;
+    } else {
+      inverse[value] = key;
+      values.push(value);
+    }
```

Comment:
no need to introduce `values` variable. 
```suggestion
    if (inverse.hasOwnProperty(value)) {
      return null;
    }
    
    inverse[value] = key;
```